wkcuber/wkcuber/convert_nifti.py

```diff
@@ -196,7 +196,6 @@ def convert_nifti(
     # Writing wkw compressed requires files of shape (shard_size, shard_size, shard_size)
     # Pad data accordingly
     padding_offset = shard_size - np.array(cube_data.shape[1:4]) % shard_size
-    padding_offset = (0, 0, 0)
```

Comment:
No idea what this what supposed to accomplish :shrug: Looks like an oversight after a refactoring to me (and mypy complained) which is why I removed it.

Reply:
@philippotto It seems like you wrote this code. Do you remember whether you deactivated the padding deliberately?

Reply:
I think this line simply slipped through while refactoring. Removing it should be fine in my opinion :+1: